Makefile

```diff
@@ -48,6 +48,8 @@ functests-on-ci: setup-test-cluster setup-build-index-image feature-deploy featu
 
 functests-on-ci-no-index-build: setup-test-cluster feature-deploy feature-wait functests
 
+feature-deploy-on-ci: setup-test-cluster setup-build-index-image feature-deploy feature-wait
```

Comment:
we have `functests-on-ci` please change that one to 

functests-on-ci: feature-deploy-on-ci functests

so we can reuse